src/main/java/uk/ac/ebi/ena/webin/cli/service/AnalysisService.java

```diff
@@ -64,24 +70,33 @@
     getAnalysis( String analysisId, String userName, String password, boolean test )
     {
         RestTemplate restTemplate = new RestTemplate();
```

Comment:
I think we should continue to use setErrorHandler and improve the NotFoundErrorHandler. See previous comment.